website/package.json

```diff
@@ -72,5 +72,12 @@
   },
   "ember": {
     "edition": "octane"
+  },
+  "dependencies": {
```

Comment:
in practice it's all the same difference, but for consistency I think it would make sense to keep these in `devDependencies`

Reply:
if it's OK I would move/review them in a follow-up PR to avoid conflicts with other PRs I have in progress